ceramic-config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "anchor": {
+    "anchor-service-url": "https://cas-qa.3boxlabs.com",
+    "ethereum-rpc-url": "https://goerli.infura.io/v3/b6685df41e1647c4be0046dfa62a020b"
+  },
+  "http-api": {
+    "cors-allowed-origins": [".*"]
+  },
+  "ipfs": {
+    "mode": "bundled"
+  },
+  "logger": {
+    "log-level": 0,
+    "log-to-files": true
+  },
+  "metrics": {
+    "metrics-exporter-enabled": false,
+    "metrics-port": 9090
+  },
+  "network": {
+    "name": "dev-unstable"
+  },
+  "node": {},
+  "state-store": {
+    "mode": "fs",
+    "local-directory": "/Users/stephaniehuynh/.ceramic/statestore/"
+  },
+  "indexing": {
+    "db": "sqlite:///Users/stephaniehuynh/.ceramic/indexing.sqlite",
```

Comment:
same question, do we need to use a different directory since there may not be a `stephaniehuynh` user on all machines where we may want to run this?  Can we use a directory under `/tmp/`?